third_party/canvaskit/icu_bidi/BUILD.gn

```diff
@@ -47,6 +47,7 @@ static_library("icu_bidi") {
     "//third_party/icu/source/common/ustrtrns.cpp",
     "//third_party/icu/source/common/utf_impl.cpp",
     "//third_party/icu/source/common/utrie2.cpp",
+    "//third_party/icu/source/common/utypes.cpp",
```

Comment:
Mind explaining what these changes mean? I think I understand adding `utypes.cpp`. It's used in debug mode to show error messages.

What about `static_library` => `source_set`?

Reply:
`utypes.cpp` was because of a compile error I was getting when debugging, for the `u_errorName_skia` symbol. Changing from `static_library` to `source_set` was just something that is a holdover from when I was troubleshooting this issue. I think I can change it back.